extensions/summarize-youtube-video-with-ai/.eslintrc.json

```diff
@@ -1,4 +1,9 @@
 {
   "root": true,
-  "extends": ["@raycast"]
-}
\ No newline at end of file
+  "extends": ["@raycast"],
```

Comment:
Can we keep as it was?

Reply:
@pernielsentikaer yea, I saw the check failing. I'm used to not have semicolons and some other QoL checks but I'll adjust.

Reply:
It's just so we keep the same with all extensions 🙂

Reply:
Got it! I reset the config and checks are green now 😊